astra-sim/network_frontend/ns3/entry.h

```diff
@@ -0,0 +1,176 @@
+#undef PGO_TRAINING
+#define PATH_TO_PGO_CONFIG "path_to_pgo_config"
+
+#include "common.h"
+#include "ns3/applications-module.h"
+#include "ns3/core-module.h"
+#include "ns3/error-model.h"
+#include "ns3/global-route-manager.h"
+#include "ns3/internet-module.h"
+#include "ns3/ipv4-static-routing-helper.h"
+#include "ns3/packet.h"
+#include "ns3/point-to-point-helper.h"
+#include "ns3/qbb-helper.h"
+#include <fstream>
+#include <iostream>
+#include <ns3/rdma-client-helper.h>
+#include <ns3/rdma-client.h>
+#include <ns3/rdma-driver.h>
+#include <ns3/rdma.h>
+#include <ns3/sim-setting.h>
+#include <ns3/switch-node.h>
+#include <time.h>
+#include <unordered_map>
+
+using namespace ns3;
+using namespace std;
+
+
+struct task1 {
+  int src;
+  int dest;
+  int type;
+  int count;
+  void *fun_arg;
+  void (*msg_handler)(void *fun_arg);
+  double schTime; // in sec
+};
+
+std::map<std::pair<int, std::pair<int, int>>, int> sender_src_port_map;

```

Comment:
Please improve the consistency in your code. You've used both std::map and map interchangeably. There's no need to prepend std:: since you've already added "using namespace std"